specification/schemas/InvoiceCustoms.json

```diff
@@ -0,0 +1,43 @@
+{
+  "type": "object",
```

Comment:
You can make this schema extend the previous schema. That way we won't have to maintain the overlapping fields in two places.

[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)